src/test/java/com/google/api/codegen/gapic/testdata/ruby/ruby_library.baseline

```diff
@@ -4970,311 +4970,882 @@ Gem::Specification.new do |gem|
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
 # DO NOT EDIT! This is a generated sample ("RequestPaged",  "odyssey")
-# FIXME: import section
-# FIXME: sample function declaration
-# FIXME: initialize client
-
-names_element = "Odyssey"
-names = [names_element]
-shelves_element = "Classics"
-shelves = [shelves_element]
-
-# Iterate over results one page at a time.
-library_service_client.find_related_books(names, shelves).each_page do |page|
-  # Process each page at a time.
-  page.each do |element|
-    # Process element.
+
+require "library"
+
+# [START sample]
+
+# Testing calling forms
+def sample_find_related_books()
+  # [START sample_core]
+  # Instantiate a client
+  client = Library.new(version: :v1)
+
+  names_element = "Odyssey"
+  names = [names_element]
+  shelves_element = "Classics"
+  shelves = [shelves_element]
+
+  # Iterate over results one page at a time.
+  client.find_related_books(names, shelves).each_page do |page|
+    # Process each page at a time.
+    page.each do |element|
+      book = element;
+      puts "Here's a related book: #{book}"
+    end
   end
+
+  # [END sample_core]
+end
+# [END sample]
+
+
+require "optparse"
+
+if $0 == __FILE__
+  sample_find_related_books()
 end
-# FIXME: response handling
-# FIXME: main function
 ============== file: samples/get_big_book_long_running_request_async_wap.rb ==============
-//FIXME: license section
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
 
 # DO NOT EDIT! This is a generated sample ("LongRunningRequestAsync",  "wap")
-# FIXME: import section
-# FIXME: sample function declaration
-# FIXME: initialize client
 
-formatted_name = Library::V1::LibraryServiceClient.book_path(shelf, "War and Peace")
+require "library"
+
+# [START hopper]
 
-operation = library_service_client.get_big_book(formatted_name) do |op|
-  raise op.results.message if op.error?
-  op_results = op.results
-  # Process the results.
+# Testing calling forms
+def sample_get_big_book(shelf)
+  # [START hopper_core]
+  # Instantiate a client
+  client = Library.new(version: :v1)
+
+  formatted_name = Library::V1::LibraryServiceClient.book_path(shelf, "War and Peace")
+
+  operation = client.get_big_book(formatted_name) do |op|
+    raise op.results.message if op.error?
+    response = op.results
+    # Testing iterating over map fields when both key and value are specified.
```

Comment:
Could I get a newline between (getting the response) and (processing the response)?

```suggestion

    # Testing iterating over map fields when both key and value are specified.
```